docs/About_Sherpa/index.rst

```diff
@@ -0,0 +1,130 @@
+In other words, KnowledgeOps *shifts further left* into the traditionally "human" stages of the workflows like "interpretation" and "synthesization" of knowledge.
+KnowledgeOps aims to significantly improve an organizations ability to handle its knowledge for internal and external purposes.
+
+*Is this too good to be true?*
+
+Until very recently, yes! 
+However, With the remarkable progress we have seen in the past few years where text based generative models started to perform on par with humans on language and tool use skills as well simpler reasoning abilities, all of a sudden there could be a way forward!
+Large Language Models (LLMs), specifically, showed a great deal of competence in a few areas that are critical in handling knowledge more effectively:
+
+- Language Skills: LLMs are very good at understanding and syntheszing language (at least for high resource languages like English, and common programming languages)
+- Tool Use: Given that LLMs are good at understanding and generating formal language (read code), they have remarkable ability to use other tools to expand their capablities by calling appropriate APIs.
+- Reasoning Skills: Although still relatively primitive, but LLMs demonstrate the ability to reason (mostly the types of reasoning that are more likely to appear in the text of their training data)  
+
+LLM Agents
+^^^^^^^^^^
+
+.. image:: imgs/multi_agent_loop.png
+  :width: 800
+  :alt: LLM Multi-agent Loop
+
+One of the most interesting trends of 2023 has been the "LLM Agents".
+These are systems that use various LLM skills to create workflows that mimic an information / data intensive one that is usually manual but can be "automated" with collaborating "agents".
+In other words, these systems devise the steps of the workflow you go through as you work towards an objective like decomposition of the main task into subtasks, planning and organization, reflection and reptition, and more to work iteratively towards a desired objective.
+This could be writing the litariture review section of a research paper, or drafting a blog post, or even carrying out a bunch of steps for creating and scheduling social media posts.
+
+Introducing SHERPA
+^^^^^^^^^^^^^^^^^^
+
+.. image:: imgs/sherpa_thinking_companion.png
+  :width: 800
+  :alt: SHERPA thinking companion
+
+In the past few years at aggregate intellect we have been focused on using AI to make interactions with documented "knowledge" more efficient and effective.
+
+Some of the pricinples we have been following in designing and building our software systems are:
+
+- These systems have to augment humans' ability to do complex tasks, rather than replacing them
+- These systems have to facilitate multiplayer interactions like human-machine-human collaborations
+- These systems have to be integrated into the typical workflows as an organic player
+
+This has led us to `SHERPA`, our LLM Agent Management Library and set of Companion Apps. 
```

Comment:
Do you want it to be `sherpa` or `SHERPA`?  If the latter, the caps usually indicates it's an acronym, so people will ask what it stands for. Not sure if you defined that somewhere, I think I missed it.